lib/tasks/convert_application_yml_to_proper_types.rake

```diff
@@ -0,0 +1,66 @@
+namespace :convert_application_yml_to_proper_types do
+  desc 'Take application.yml input file and convert to proper type'
+  task :convert, [:filename] => :environment do |t, args|
+    VALUE_TYPES = ['float', 'string', 'boolean', 'json', 'timestamp', 'integer']
+    filename = args[:filename]
+
+    if filename
+      puts "Making backup copy of '#{filename}' to '#{filename}.bak'"
+      FileUtils.copy(filename, filename+'.bak')
+      content=''
+      add_spaces = false
+      File.open(filename+'.tmp', 'w') { |newfile|
```

Comment:
two style things:
1. spaces around `content`
2. I think we prefer the `do...end` syntax for multiine blocks

It looks like Rubocop is not running on this file because it ignores `lib/taks/*.rake`, can we add this one to be included?

```suggestion
      content = ''
      add_spaces = false
      File.open(filename+'.tmp', 'w') do |newfile|
```

Reply:
kk - 1 and 2 are done.

Where are you seeing the ignore rakes? I'm looking in .rubocop.yml and see below, looks like it should be ok?
```

AllCops:
  Exclude:
    - 'bin/**/*'
    - 'db/primary_migrate/*'
    - 'db/schema.rb'
    - 'db/worker_jobs_migrate/*'
    - 'db/worker_jobs_schema.rb'
    - 'lib/tasks/create_test_accounts.rb'
    - 'node_modules/**/*'
    - 'tmp/**/*'
    - 'vendor/**/*'
    - 'public/**/*'
  TargetRubyVersion: 2.7
  TargetRailsVersion: 6.0
  UseCache: true
  DisabledByDefault: true
  SuggestExtensions: false


```

Reply:
Ah maybe it was just block length:
https://github.com/18F/identity-idp/blob/7ad3feab24f6f9e0e45224d9e9be9458c0a6a648/.rubocop.yml#L605

Reply:
Kk updated.